packages/fast-development-site-react/src/components/site/action-bar.tsx

```diff
@@ -8,6 +8,11 @@ import { glyphBuildingblocks, glyphExamples, glyphPage } from "@microsoft/fast-g
 import ComponentViewToggle from "./component-view-toggle";
 import manageJss, { ComponentStyles, ICSSRules, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
 
+export enum actionEnum {
```

Comment:
I think we've been capitalizing enums throughout all projects, should we do so here? Or, is there a principle for not doing so here?

Reply:
Also seeing this with frameworkEnum